app/src/main/java/org/p2p/wallet/solend/ui/deposits/SolendUserDepositsFragment.kt

```diff
@@ -61,4 +65,11 @@ class SolendUserDepositsFragment :
             tokensRecyclerView.isVisible = !isLoading
         }
     }
+
+    override fun onAddMoreClicked(token: SolendDepositToken.Active) {
```

Comment:
is there any implementation, if not maybe left TODO here

Reply:
no, the action is here - https://github.com/p2p-org/p2p-wallet-android/pull/849
this needs to be merged asap